devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -94,12 +96,20 @@ private getDOORLOCK_ATTR_MIN_PIN_LENGTH() { 0x0018 }
 private getDOORLOCK_ATTR_SEND_PIN_OTA() { 0x0032 }
 private getALARM_ATTR_ALARM_COUNT() { 0x0000 }
 private getALARM_CMD_ALARM() { 0x00 }
+private getIAS_ATTR_ZONE_STATUS() {0x0010}
```

Comment:
ZoneStatus is attribute ID 0x0002. Attribute ID 0x0010 is the CIE address.

Reply:
ok